tools/ci.sh

```diff
@@ -759,6 +759,7 @@ function ci_zephyr_build {
     docker exec zephyr-ci west build -p auto -b frdm_k64f
     docker exec zephyr-ci west build -p auto -b mimxrt1050_evk
     docker exec zephyr-ci west build -p auto -b nucleo_wb55rg # for bluetooth
+    docker exec zephyr-ci west build -p auto -b beagleconnect_freedom
```

Comment:
I don't think we should add this board to CI, because the zephyr CI here already takes a long time (and not all boards are tested anyway).

Reply:
@dpgeorge Where are the images present https://micropython.org/download/ built?

Reply:
They are build by the scripts in `tools/autobuild`.

But currently zephyr is not included in this, we don't provide pre-built firmware for any zephyr board.

We can start to do that, for boards that are known to work well with MicroPython and which have been tuned to enable the correct features.

Reply:
Sure, Let's stick to board configs for now and after some community testing, we can revisit providing pre-built firmware in the future.